### ISSUE TITLE
Dependency on gradle 3.1.0 fails

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mbgl-ci-images#18]
The default project generated by Android Studio 3.1 will fail in CI on the current android-ndk-r16b image. It took me several hours to figure out that I had to change `build.gradle`:

Works:

```
buildscript {
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.0.1'
    }
}
```

Doesn't work:

```
buildscript {
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.1.0'
    }
}
```

The latter [fails](https://circleci.com/gh/mapbox/mapbox-bindgen/48) with the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     build-tools;27.0.3 Android SDK Build-Tools 27.0.3
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
  Alternatively, to transfer the license agreements from one workstation to another, see http://d.android.com/r/studio-ui/export-licenses.html
  
  Using Android SDK: /android/sdk
```

**Comment by zugaldia** [comment]
Good catch, thanks @jfirebaugh for the research!

cc: @tobrun fyi.

**Comment by tobrun** [comment]
I was initially waiting for the stable release of ndk 17 [here](https://github.com/mapbox/mbgl-ci-images/issues/16) to update the image but capturing from the [ndk-release-schedule](https://github.com/android-ndk/ndk/wiki#release-schedule) that this will only happen in May. I will work on updating the image.

**Comment by kkaefer** [comment]
We're intentionally removing the license agreement again to prevent builds from installing new software.

**Comment by tobrun** [comment]
available with `mbgl/feb0443038:android-ndk-r17`
